src/Workspaces/Core/Portable/Rename/RenameLocations.cs

```diff
@@ -92,7 +92,7 @@ internal sealed partial class RenameLocations
                         if (result.HasValue && result.Value != null)
                         {
                             var rehydrated = await TryRehydrateAsync(
-                                solution, fallbackOptions, result.Value, cancellationToken).ConfigureAwait(false);
+                                solution, symbol, fallbackOptions, result.Value, cancellationToken).ConfigureAwait(false);
```

Comment:
this is the core change. we are already passing the symbol in, in order to make the oop call.  so no need to try to have dehydrate/rehydrate that exact same symbol in its response when we can just literally use the same one we started with.